site/content/blog/2020-05-02-a-growing-community.md

```diff
@@ -0,0 +1,42 @@
+---
+title: A growing community
+description: The Svelte community keeps on growing. Here's a summary of what's been happening over the past year.
+author: Johan Ronsse
+authorURL: https://twitter.com/wolfr_2_
+---
+
+It's been a year since the release of Svelte 3, and over the year, a lot has happened in terms of community. In this post, I am collecting some highlights of the past year, from a community perspective.
+
+This graph is only one slice of the pie, but here is the interest in Svelte as measured by Github stars over time.
+
+<img src="/blog/graph.png" style="max-width: 100%;">
```

Comment:
This image isn't part of this PR.

That said, I'm personally not particularly fond of even having a graph of the number of stars over time. We're not trying to sell a product to investors here. I'd say just let the bullet points below do the talking.

Reply:
OK!